content/docs/components-and-props.md

```diff
@@ -80,24 +79,24 @@ ReactDOM.render(
 
 [](codepen://components-and-props/rendering-a-component)
 
-Let's recap what happens in this example:
+Recapitulemos lo que sucede en este ejemplo:
 
-1. We call `ReactDOM.render()` with the `<Welcome name="Sara" />` element.
-2. React calls the `Welcome` component with `{name: 'Sara'}` as the props.
-3. Our `Welcome` component returns a `<h1>Hello, Sara</h1>` element as the result.
-4. React DOM efficiently updates the DOM to match `<h1>Hello, Sara</h1>`.
+1. Llamamos a `ReactDOM.render()` con el elemento `<Welcome name="Sara" />`.
+2. React llama al componente `Welcome` con `{name: 'Sara'}` como "props".
+3. Nuestro componente `Welcome` devuelve un elemento `<h1>Hello, Sara</h1>` como resultado.
+4. React DOM actualiza eficientemente el DOM para que coincida con `<h1>Hello, Sara</h1>`.
 
->**Note:** Always start component names with a capital letter.
->
->React treats components starting with lowercase letters as DOM tags. For example, `<div />` represents an HTML div tag, but `<Welcome />` represents a component and requires `Welcome` to be in scope.
->
->You can read more about the reasoning behind this convention [here.](/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized)
+> **Note:** Siempre comienza los nombres de componentes con una letra mayúscula
+> 
+> React trata los componentes que empiezan con letras minúsculas como etiquetas del DOM. Por ejemplo, `<div />` representa una etiqueta div HTML pero `<Bienvenido />` representa un componente y requiere que `Bienvenido` esté al alcance.
```

Comment:
`scope` is a technical term, I'm not sure whether we should be translating it. 

What do you think?

Reply:
Im confused about the meaning in the context. So should it be:
` requiere que `Bienvenido` esté en scope`??

Reply:
If I'm not mistaken, when it says that it needs to be in scope it refers to it being an accessible variable within the parent's component code. 
This can be achieved by having the child component declared on the same file and scope, or using imports/requires. 
This is of course related to [scope in Javascript](https://developer.mozilla.org/en-US/docs/Glossary/Scope).

As to how the translation could go, I believe your latter suggestion is correct.

By the way, I see that you translated the name of the component to "Bienvenido", but I think I also saw it as "Welcome" a few lines above that. Is that under control?